javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,24 @@
+var $char = document.querySelectorAll('span');
+var counter = 0;
+var currentChar = $char[counter].textContent;
+var $again = document.querySelector('.hide');
+var $button = document.querySelector('button');
+
+document.addEventListener('keydown', function (event) {
+  var key = event.key;
+  if (key === currentChar) {
+    $char[counter].className = 'green';
+    if (counter === $char.length - 1) {
+      $again.className = 'show';
+      $button.addEventListener('click', function (event) {
+        location.reload();
```

Comment:
Hey Ariel, nice work on the reset functionality! The one thing to touch on here is that you want to avoid resetting the application by using reload, instead, I would recommend creating a `reset` function which resets the application back to the original state without refreshing the page.

Reply:
Okay, thank you!